app/models/account.rb

```diff
@@ -230,18 +230,16 @@ def fields_attributes=(attributes)
     fields     = []
     old_fields = self[:fields] || []
 
-    if attributes.is_a?(Hash)
-      attributes.each_value do |attr|
-        next if attr[:name].blank?
+    Array.wrap(attributes).each do |attr|
+      next if !attr.is_a?(Hash) || attr[:name].blank?
```

Comment:
we should throw a validation error here instead of passing silently, to resolve #9611

Reply:
It raises error now when it fails the validation.

Reply:
```
     if !attr.is_a?(Hash) || attr[:name].blank?
        raise ArgumentError, 'One of custom field attributes has invalid format'
      end
```